impeller/compiler/compiler.cc

```diff
@@ -36,6 +37,43 @@ static CompilerBackend CreateMSLCompiler(const spirv_cross::ParsedIR& ir,
       spirv_cross::CompilerMSL::Options::make_msl_version(1, 2);
   sl_compiler->set_msl_options(sl_options);
 
+  auto floats =
```

Comment:
Some comments about why this is being done here could be helpful.

Reply:
Done